approx_dates/models.py

```diff
@@ -6,6 +6,15 @@
 
 import six
 
+class classproperty(object):
```

Comment:
This is quite a subtle decorator, so it would be good to add a comment explaining in just a sentence what it's for. Also, it looks as if this was taken from [a StackOverflow answer](http://stackoverflow.com/a/13624858/223092) so it would be good to link to that from a comment.